src/_data/toc/graphql.yml

```
@@ -265,6 +265,11 @@ pages:
           edition: ee-only
           exclude_versions: ["2.3"]
 
+        - label: createGiftRegistry mutation
+          url: /graphql/mutations/create-gift-registry.html
+          edition: b2b-only
```

Please, correct me if I'm wrong, but this feature is available for ee as well

```suggestion
          edition: ee-only
```